vochain/indexer/db/models.go

```diff
@@ -25,11 +26,10 @@ type Block struct {
 type Process struct {
 	ID                 types.ProcessID
 	EntityID           types.EntityID
-	StartBlock         int64
-	EndBlock           int64
 	StartDate          time.Time
 	EndDate            time.Time
-	BlockCount         int64
+	ManuallyEnded      bool
+	BlockCount         sql.NullInt64
```

Comment:
Do we really need to use this type ?

Reply:
You are probably seeing the diff changes from the feature branch to the main. This change is not from this PR.

Reply:
Seems so, ignore then my last comment, regarding mixing changes. Sorry about that.